src/helpers/titles.ts

```diff
@@ -0,0 +1,7 @@
+export const clipTitle = (title: string, chars: number = 15) => {
+  if (title.length <= chars) {
```

Comment:
can it be done with just css clip?

Reply:
Overflow hidden, clip, but why do I feel bad about that? Can you please probide additional arguments to why CSS should be used for such action rather than js?